tensorflow/contrib/framework/BUILD

```diff
@@ -24,7 +27,42 @@ py_library(
         "python/ops/prettyprint_ops.py",
         "python/ops/variables.py",
     ],
+    data = [
+        ":python/ops/_variable_ops.so",
```

Comment:
I find if `data` is removed, `tensorflow/contrib/framework/python/ops/variables_test.py` will failed with `ImportError: cannot import name gen_variable_ops`